spec/mdn/mdn.mdown

```diff
@@ -0,0 +1,278 @@
+- **error**: `String[]|null` (server-set)
+  Additional information in the form of text messages when the "error" disposition modifier appears.
+- **extensionFields**: `String[String]|null` (server-set)
+  Object where keys are extension-field names and values are extension-field values.
+
+  A **Disposition** object has the following properties:
+
+  - **actionMode**: `String`
+    This MUST be one of the following strings: "manual-action" / "automatic-action"
+  - **sendingMode**: `String`
+    This MUST be one of the following strings: "MDN-sent-manually" / "MDN-sent-automatically"
+  - **type**: `String`
+    This MUST be one of the following strings: "deleted" / "dispatched" / "displayed" / "processed"
+
+  See [@!RFC8098] for the exact meaning of these different fields.
+
+# Methods added to the EmailSubmission object
+
+## EmailSubmission/sendMDN
+
+The EmailSubmission/sendMDN method generates and sends an [@!RFC5322] message from an MDN object.
+
+It takes the following arguments:
+
+- **accountId**: `Id`
+  The id of the account to use.
+- **mdns**: `String[MDN]`
+  A map of creation id (client specified) to MDN objects
+
+If the *forEmailId*, *subject*, *textBody*, *reportingUA*, *disposition* properties are invalid (e.g. missing, wrong type, id not found), the submission creation is rejected with a standard `invalidProperties` SetError and no email is sent. Any other error usually sent by "EmailSubmission/set" for **create** can be returned by this method.
+
+The client SHOULD NOT issue a sendMDN request if the message has the `$MDNSent` keyword set. In this case, the server MUST reject the submission with a standard `forbiddenToSend` SetError.
+
+When sending the MDN, the server is in charge of generating the *originalRecipient*, *finalRecipient* and *originalMessageID* fields accordingly to the [@!RFC8098] specification.
+
+The response has the following arguments:
+
+- **accountId**: `String`
+  The id of the account used for this call.
+- **created**: `String[EmailSubmission]`
+  A map of creation id (client-specified) to an email sent from the referenced properties. The returned EmailSubmission is similar to a call to a standard "EmailSubmission/set" with a *create* parameter.
+- **notCreated**: `String[SetError]`
+  A map of creation id to a SetError object for each Email that failed to be sent. The possible errors are defined above.
+
+For each `forEmailId` whose EmailSubmission where created, the server MUST add a `$MDNSent` keyword to the email.
+
+## EmailSubmission/parseMDN
+
+This method allows you to parse blobs as [@!RFC5322] messages to get MDN objects. This can be used to parse and get detailed information about blobs referenced in the *mdnBlobIds* of the EmailSubmission object, or any email the client could expect to be an MDN.
+
+The *forEmailId* property can be null or missing if the *originalMessageID* property is missing or not referencing an existing email.
```

Comment:
Does it make sense to parse it at all?

Reply:
Sorry, I don't understand your question

Reply:
I mean, If the MDN doesn't match any message you have, why would you want to parse it?

Reply:
because you could have lost the sent mail matching it, but still want to be informed of received MDN. Fields in the MDN can still be relevant to the user.